src/display.c

```diff
@@ -64,6 +64,9 @@ open_external_viewer(const char *argv[], const char *dir, bool silent, bool conf
 				fprintf(stderr, "%s", notice);
 			fprintf(stderr, "Press Enter to continue");
 			getc(opt_tty);
+			opt_tty = freopen(NULL, "r", opt_tty);
```

Comment:
If I read `freopen(3)` the mode should be `"w+"` to match how it is initially opened in pager mode: https://github.com/jonas/tig/blob/1f757f6579f6c56b68bbd68c6896a3c1f204b859/src/display.c#L557